projects/compiler/test/test-runner.js

```diff
@@ -62,9 +63,27 @@ class TestRunner {
   async _runCompilerTest(bin, testFile, args = [], env = {}) {
     const testFilePath = `${__dirname}/${testFile}`
 
+    async function buildAndRunTestBin(testFilePath, compilerBin, args, env, isJsTarget) {
```

Comment:
this is nasty but it will eventually be replaced by a test runner written in abra, so i'm fine with it for now